sqlglot/dialects/presto.py

```diff
@@ -345,6 +348,17 @@ class Parser(parser.Parser):
         FUNCTION_PARSERS = parser.Parser.FUNCTION_PARSERS.copy()
         FUNCTION_PARSERS.pop("TRIM")
 
+        PROPERTY_PARSERS: t.Dict[str, t.Callable] = {
+            **parser.Parser.PROPERTY_PARSERS,
+            "SECURITY": lambda self: self._parse_security(),
+        }
+
+        def _parse_security(self) -> t.Optional[exp.SecurityProperty]:
+            if self._match_set((TokenType.DEFINER, TokenType.INVOKER)):
```

Comment:
You can use `self._match_texts(("DEFINER", "INVOKER"))` instead (see comment re: getting rid of tokens).